sdk/tests/types/fixtures/basic_block_tagged_data_payload.json

```diff
@@ -0,0 +1,42 @@
+{
+    "block":{
+       "header":{
+          "protocolVersion":3,
+          "networkId":"0",
+          "issuingTime":"1701088670011833888",
+          "slotCommitmentId":"0x3a1e3b617060146e0362361a4b752833186108395f3b2b3d3e6c655e287d70767ea58d2a",
+          "latestFinalizedSlot":500,
+          "issuerId":"0x17432c5a7a672503480241125e3952414a7a320441080c624c264b004e09614a"
+       },
+       "body":{
+          "type":0,
+          "strongParents":[
+             "0x2674252933745d134c1369621e366b0e63171c3e0b0c7b654009640b1316236469711226",
+             "0x382854566f5e26533c391e2e4962434d13363a037e07563c522527566d536e165d0b0371",
+             "0x523950126960352d2c5e35081d3f2128125f2b77502f4b1415440e3b260371216c1c645a",
+             "0x525e5a4d011d6359253e3e6e32717a2f0b354478154e1f4426345608234e044e5b62187c",
+             "0x6d1859664875241d0c6a4c55146f1d491c0a1f5a090e785c4a7b691977730e75622a5d41",
+             "0x742c2e2d080a42165c52732b4e48245e795e61240b7832625d1c0f037c5937191f510f07"
+          ],
+          "weakParents":[
+             
+          ],
+          "shallowLikeParents":[
+             
+          ],
```

Comment:
These entries shouldn't exist if empty?

Reply:
I copy paste from the TIP, I'll comment there